platform/app/src/utils/isSeriesFilterUsed.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * This function is used to check if the filter is used. Its intend is to
+ * warn the user in case of link with a SeriesInstanceUID was called
+ * @param instances
+ * @returns
+ */
+export default function isSeriesFilterUsed(instances, filters) {
```

Comment:
I am still confused as to what this function is intended to do based on its name and JS docs. By the now (corrected) logic, it seems to check if the series for the given `instances` is to be "kept" (return true) or filtered out (return false). If so then consider naming this function `isSeriesNotFiltered` or maybe use `isSeriesFiltered` and reverse when true and false are returned.

Furthermore, should this URL show a warning? 
http://localhost:3000/viewer?StudyInstanceUIDs=1.3.6.1.4.1.25403.345050719074.3824.20170125095258.1&seriesinstanceuid=non.existent,1.3.6.1.4.1.25403.345050719074.3824.20170125095258.2
I had expected it to. But maybe I do not undertand the use case.